src/quote/Quote.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Quote construct a framework for Attesters to make an offer for building a [[Claim]] on a [[CTYPE]] in which it includes a price and other terms & conditions upon which a claimer can agree.
```

Comment:
Typo/Fixes:
- constructs
- Quote--> [[Quote]] ?

Reply:
Thank you!